deselected_tests.yaml

```diff
@@ -145,3 +145,5 @@ deselected_tests:
   - inspection/tests/test_partial_dependence.py::test_partial_dependence_dataframe[features-string-None-estimator-brute]
   - ensemble/tests/test_bagging.py::test_estimators_samples_deterministic
 
+  # Eigenvalues in the last component aren't equal because of different implementations of PCA
```

Comment:
write here for what reason the test failed. diverge accuracy?

Reply:
Some values in `PCA.components_` (in the last component) are not equal. Also, results are not stable

Reply:
How much on average?

Reply:
0.6 on average for absolute error in this test

Reply:
Add this info to comments